tests/fast-integration/telemetry-test/telemetry-test.js

```diff
@@ -89,7 +111,8 @@ describe('Telemetry Operator tests', function() {
 
   it('Should create valid LogPipeline with HTTP output plugin', async () => {
     await k8sApply(logPipelineCR, telemetryNamespace);
-    await waitForDaemonSet(fluentBitName, telemetryNamespace, 30000);
+    await waitForLogPipelineStatusCondition('logpipeline-test', 'Pending', 20000);
```

Comment:
I am wondering if there is a chance that we make this call too late and miss the `Pending` state. This could potentially make the test flaky.

Reply:
We can rely on the fact that the transition from Pending to Running takes time if in the case of a one-node cluster. We actually caught a bug in our code when the transition was happening too fast. But I agree, let us observe how "stable" this assumption is.